corehq/apps/locations/util.py

```diff
@@ -184,6 +185,7 @@ def _loc_type_dict(self, loc_type):
             uncategorized_keys.update(uncategorized_data.keys())
 
             loc_dict = {
+                'location_id': loc.location_id,
```

Comment:
Should `loc_dict` contain `location_id` irrespective of whether `include_ids` is `False`